{"uuid": "dfdad8d9-0747-4ba4-8e85-507549696d03", "children": ["3db20895-8247-4075-ab8e-769514d43b13", "97052f7e-ed15-4849-9661-3f7cd13a47d7"], "befores": [{"name": "_Class__pytest_setup_class", "status": "broken", "statusDetails": {"message": "IndexError: list index out of range\n", "trace": "  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 788, in call_fixture_func\n    res = next(it)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\_pytest\\python.py\", line 703, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\_pytest\\python.py\", line 634, in _call_with_optional_argument\n    func(arg)\n  File \"F:\\python\\zrapiautotestv1.0\\TestCase\\StockQuotation\\OptionalStock\\test_stick_optional_stock.py\", line 17, in setup_class\n    cls._id = {\"id\": str(id[0].get('_id'))}\n"}, "start": 1593328815600, "stop": 1593328815660}], "start": 1593328815600, "stop": 1593328815672}